src/dbus/common/types.hpp

```diff
@@ -703,6 +703,22 @@ struct InfraLinkInfo
     uint32_t    mGlobalUnicastAddresses; ///< The number of global unicast addresses on the infra network interface.
 };
 
+struct TrelInfo
+{
+    struct TrelCounters
```

Comment:
Shall we call it `TrelPacketCounters` instead? Also it may not be necessary to have these fields as a struct.

Reply:
done

Reply:
keep the struct to be consistent with other counters in this file.